content/en/apps/guides/messaging/africas-talking.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Africa's Talking SMS Aggregator
+title: Africa’s Talking SMS Aggregator
```

Comment:
Are backticks better?

Reply:
Well, my old boss would say that `’` is the proper apostrophe and `'` is, at least in the US, a [foot mark](https://practicaltypography.com/foot-and-inch-marks.html) ;)  Also, isn't that just a normal tick, not a backtick!?!  Too many rabbit holes, sorry ;)

In this case the `'` got rendered by `hugo` into `&#39` which `muffet` insisted was a URL fragment where as  `’` was something without the `#` character.  A very fragile change, I know. I should probably file a ticket against `muffet`, but after my bouts in the ring with FF and enketo, I was gonna sit this one out.

I called this out in the ticket with the list of links (see "Results - Special Fixes" on #260).

please unresolve if this doesn't resolve this!!